src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllContext.cs

```diff
@@ -85,17 +85,22 @@ public partial class FixAllContext
             Document document,
             CodeFixProvider codeFixProvider,
             FixAllScope scope,
-            string codeActionEquivalenceKey,
+            string? codeActionEquivalenceKey,
             IEnumerable<string> diagnosticIds,
             DiagnosticProvider fixAllDiagnosticProvider,
             CancellationToken cancellationToken)
-            : this(new FixAllState(null, document, codeFixProvider, scope, codeActionEquivalenceKey, diagnosticIds, fixAllDiagnosticProvider),
+            : this(new FixAllState(
+                    fixAllProvider: null,
+                    document ?? throw new ArgumentNullException(nameof(document)),
```

Comment:
💡 Use `!!` instead

Reply:
I tried but getting syntax formatter warnings :(.

Reply:
![image](https://user-images.githubusercontent.com/41759/154331322-3dc21129-f63d-4359-a12d-072818206341.png)

Reply:
Also interesting - there can't be  whitespace between the identifier and `!!` :-o
![image](https://user-images.githubusercontent.com/41759/154331516-18809632-a545-43e8-9712-0f5307197cb1.png)

Reply:
Filed https://github.com/dotnet/roslyn/issues/59580